src-deprecated/PeclDispatcher.php

```
@@ -0,0 +1,32 @@
+ *                    Use proxy-based AOP with $aspect->bind() + newInstance() instead.
+ *                    See https://github.com/ray-di/Ray.Aop/issues/242 for details.
+ */
+final class PeclDispatcher
+{
+    /**
+     * @param mixed ...$args Accepts any arguments for backward compatibility
+     *
+     * @throws LogicException Always throws as PECL extension is no longer supported
+     * @deprecated 2.19.0 Use proxy-based AOP instead
+     */
+    public function __construct(mixed ...$args)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)